.Makefile.mk

```diff
@@ -31,6 +31,9 @@ TEMPLATEPATH=../.Makefile.template
 ### default target ###
 DEFAULT += help
 
+# header and source files proccessed by format and check
+FILES=$(shell cd ../; find -regex '.*\.\(c\|h\)' -not -path *build*)
```

Comment:
why is this variable not in the variables section?
why determine files here and not inside the container?
why the `cd ..` when using `find`?

Reply:
if i tried to solve it in find, sometimes no files where returned. unfortunately it wasn't a deterministic problem.

Reply:
It was initially in the variables sections, seems to be moved during rebase. will fix that